source/includes/_api.md

```diff
@@ -3,10 +3,10 @@
 > Digital Goods API Endpoint
 
 ```
-https://api.satoshipay.io/v1/
+https://api.satoshipay.io/v2/
 ```
 
-The Digital Goods API allows developers to interact with SatoshiPay using HTTP REST calls and JSON. Digital goods merchants communicate with the API in order to register individual goods for sale &ndash; either directly or through plugins and libraries provided by SatoshiPay or 3rd parties. The digital goods merchant hosts complementary [HTTP Endpoints](#html-endpoints) that deliver the goods to the user.
+The Digital Goods API allows developers to interact with SatoshiPay using HTTP REST calls and JSON. Digital goods merchants communicate with the API in order to register individual goods for sale—either directly or through plugins and libraries provided by SatoshiPay or 3rd parties. The digital goods merchant hosts complementary [HTTP Endpoints](#html-endpoints) that deliver the goods to the user.
```

Comment:
In case anyone is curious: https://en.wikipedia.org/wiki/Dash#Em_dash

Reply:
Fair enough, but I think there should be a space before and after the dash, right?

Reply:
Debatable. Some style guides say none, some say hair spaces. I can make it hair spaces if you'd prefer

Reply:
You are right and I really didn't see it coming. Looks wrong to me to not have spaces around, but look what Google even says:

<img width="728" alt="bildschirmfoto 2017-12-04 um 09 49 34" src="https://user-images.githubusercontent.com/1842462/33543776-94b1842c-d8d8-11e7-970a-852e0d746d4c.png">